src/index.js

```diff
@@ -0,0 +1,207 @@
+'use strict';
+
+// State tracking object
+const weather = {
+  temperature: 72,
+  clouds: 0,
+  condition: 'Clear',
+  city: 'Atlanta',
+  updating: false,
+};
+
+// Add options to select menu
+const skies = ['☀️', '🌤', '⛅️', '🌥', '🌦', '🌧', '🌈'];
+const skyMenu = document.getElementById('sky-selector');
+
+const createOption = (emoji) => {
+  const opt = document.createElement('option');
+  opt.textContent = emoji;
+  if (emoji === '☀️') {
+    opt.selected = true;
+  }
+  return opt;
+};
+
+for (const skyOption of skies.map(createOption)) {
+  skyMenu.appendChild(skyOption);
+}
+
+// Helper functions used in updates
+const tempClass = () => {
+  if (weather.temperature > 87) {
+    return 'hot';
+  }
+  if (weather.temperature > 77) {
+    return 'warm';
+  }
+  if (weather.temperature > 67) {
+    return 'goldilocks';
+  }
+  if (weather.temperature > 57) {
+    return 'cool';
+  }
+  return 'cold';
```

Comment:
Notice that the logic on lines 31-43, how similar the branches are of the conditional statement. We look for whether the temperature is within a range, and return a class name accordingly. Another approach to this would be to set up a list of objects like:
```javascript
const TEMP_CLASS = [ 
  { upperBound: 47, color: "cold"}, 
  { upperBound: 57, color: "cool"}, 
  { upperBound: 67, color: "goldilocks"}, 
  { upperBound: 77, color: "warm"}, 
  { upperBound: 87, color: "hot"}
];
```

Then your method would iterate through TEMP_CLASS, compare `weather.temperature` to the upperBound key and return the right class name, which would help make the function a bit more concise.

This could accommodate a scenario where you might have even more backgrounds in the future, which would prevent you from having a really lots of if statements.